### ISSUE TITLE
Explore Azure WebSiteManagement REST client library 🚀 

### ISSUE THREAD

**Issue by alexweininger** [microsoft/vscode-azurestaticwebapps#713]
Azure SDK team [has released](https://github.com/Azure/azure-sdk-for-js/pull/22044) a beta of what they call a ["REST level client" or RLC.](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md)

https://www.npmjs.com/package/@azure-rest/arm-appservice

Read [this issue](https://github.com/Azure/azure-sdk-for-js/issues/22017) to learn about what problems this new type of SDK solves for us. 

In summary, 
old package bundle size: [`940.4 kB`](https://bundlephobia.com/package/@azure/arm-appservice@11.0.0)
new package bundle size: [`38.4 kB`](https://bundlephobia.com/package/@azure-rest/arm-appservice@1.0.0-beta.1) 🚀 

They'd like to hear any feedback we have here 👉 https://github.com/Azure/azure-sdk-for-js/issues/22017
